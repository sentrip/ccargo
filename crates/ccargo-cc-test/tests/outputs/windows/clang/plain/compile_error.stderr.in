main.c:4:9: error: initializing 'int' with an expression of incompatible type 'void'
    int x = func();
        ^   ~~~~~~
main.c:5:9: error: initializing 'int' with an expression of incompatible type 'void'
    int y = func();
        ^   ~~~~~~
2 errors generated.
