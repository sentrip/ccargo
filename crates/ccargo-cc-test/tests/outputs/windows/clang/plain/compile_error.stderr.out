error: initializing 'int' with an expression of incompatible type 'void'
   --> main.c:4:9
    4 |     int x = func();
      |         ^   ~~~~~~
error: initializing 'int' with an expression of incompatible type 'void'
   --> main.c:5:9
    5 |     int y = func();
      |         ^   ~~~~~~
2 errors generated.
